addons/mail/static/src/components/messaging_menu/tests/messaging_menu_tests.js

```diff
@@ -266,6 +266,7 @@ QUnit.test('counter is taking into account failure notification', async function
     this.data['mail.notification'].records.push({
         mail_message_id: 11, // id of the related message
         notification_status: 'exception', // necessary value to have a failure
+        notification_type: 'email', // expected failure type for email message
```

Comment:
no need for the comment, it's a mistake from the past, too verbose for not enough gain